src/interface/polarplot.jl

```diff
@@ -53,11 +54,21 @@ julia> polarplot(range(0, 2π, length = 20), range(0, 2, length = 20))
 
 `Plot`, `lineplot`, `BrailleCanvas`
 """
-function polarplot(θ::AbstractVector, 𝓇::Union{Function,AbstractVector}; kw...)
+function polarplot(
+    θ::AbstractVector,
+    𝓇::Union{Function,AbstractVector};
+    𝓇lim = (0, 0),
+    kw...,
+)
     pkw, okw = split_plot_kw(; kw...)
-    𝓇 = 𝓇 isa Function ? 𝓇.(θ) : 𝓇
 
-    mr, Mr = extrema(𝓇)
+    if 𝓇lim == (0, 0)
```

Comment:
Can you use [`is_auto`](https://github.com/JuliaPlots/UnicodePlots.jl/blob/86d6813cd8311ede98da5b618163ab1f3668c11e/src/common.jl#L378) ?